src/io/github/thebusybiscuit/cscorelib2/config/Localization.java

```diff
@@ -114,6 +114,7 @@ public void setDefaultMessages(String key, String... messages) {
 	 *
 	 * @param  key The Key of those Message
 	 * @param  message The Message which this key will refer to by default
+	 * @return The message set
```

Comment:
This is not quite accurate.
It returns the previously assigned message, the given message if none was found.